apps/challenges/views.py

```diff
@@ -2830,24 +2830,33 @@ def get_aws_credentials_for_participant_team(request, phase_pk):
             - When participant_team has not participanted in challenge
             - When Challenge is not Docker based
     """
-    challenge_phase = get_challenge_phase_model(phase_pk)
-    challenge = challenge_phase.challenge
-    participant_team = get_participant_team_of_user_for_a_challenge(
-        request.user, challenge.pk
-    )
-    if not challenge.is_docker_based:
-        response_data = {
-            "error": "Sorry, this is not a docker based challenge."
-        }
-        return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+    try:
+        logging.debug(f"Retrieving challenge phase for ID: {phase_pk}")
```

Comment:
Could you please provide an explanation for why this change was added?

Reply:
@Suryansh5545 it creates a log entry, developers can access for troubleshooting and understanding the code's behavior.

the idea of logging was mentioned by @gchhablani in his pr, "https://github.com/Cloud-CV/EvalAI/pull/4261".

this was all the behaviour i tried to add in the code, kindly let me know further any.
plus this also adds the try-catch block which already @gchhablani did.

Reply:
@AishwaryJain07 i think you misunderstood it. we want to give a error message to the user as return if something goes wrong instead of a generic message. you are going in the right direction by capturing the error and sending it back as a response. but the log thing isn't required for now.

Reply:
@AishwaryJain07 We do not want logging.